operator/main.go

```diff
@@ -174,20 +179,23 @@ func run() error {
 	return nil
 }
 
-func maybeUseLegacyTLSFileLocation(mgr manager.Manager) {
+func getWebhookOptions() webhook.Options {
 	// OLM before version 0.17.0 (such as the one shipped with OpenShift 4.6) does not
 	// provide the TLS certificate/key in the location referenced by default by the controller runtime
 	// (i.e. /tmp/k8s-webhook-server/serving-certs/...).
 	// See ROX-8304 for details.
 	// If the files are missing at the default location, then we explicitly set the settings as follows
 	// to force usage of the legacy location, which is provided both by old and new OLM, but not
 	// by the "make deploy" scaffolding.
+	opts := webhook.Options{
+		Port: 9443,
```

Comment:
It's `9443` by default. Should we still specify it for transparency?

Reply:
Not sure why it was specified on line 117, but I'd like to limit the scope of this PR to the things necessary for the bump. I also think having the number here makes sense, in case someone greps for the port number.